src/test/java/com/thebois/models/world/TestWorld.java

```diff
@@ -22,7 +23,11 @@ public TestWorld(final int worldSize, final ThreadLocalRandom random) {
     @Override
     protected ITerrain[][] setUpTerrain(final int worldSize, final int seed) {
         final ITerrain[][] terrainMatrix = new ITerrain[worldSize][worldSize];
-        MatrixUtils.populateElements(terrainMatrix, Grass::new);
+        for (int y = 0; y < worldSize; y++) {
+            for (int x = 0; x < worldSize; x++) {
+                terrainMatrix[y][x] = TerrainFactory.createTerrain(TerrainType.GRASS, x, y);
+            }
+        }
```

Comment:
style

It was really clean before, can't you do
```java
MatrixUtils.populateElements(terrainMatrix, (x, y) -> TerrainFactory.createTerrain(TerrainType.GRASS, x, y));
```
or was there something that borked when doing it this way?